CMakeLists.org

```diff
@@ -0,0 +1,59 @@
+cmake_minimum_required(VERSION 3.12)
```

Comment:
We don't need this file, this is a copy of the original file and if we want to refer to the original file later, we can see it in the history or refer to the original repo we forked from. Please remove

Reply:
Removed the file